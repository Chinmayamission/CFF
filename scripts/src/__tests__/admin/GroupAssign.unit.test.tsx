import React from "react";
import { shallow, mount, render } from 'enzyme';
import {Provider} from "react-redux";
import store from "src/store";
import GroupAssign from "../../admin/GroupAssign/GroupAssign";

it('renders group assign correctly', () => {
  const wrapper = render(
    <Provider store={store}><GroupAssign /></Provider>
  );
  expect(wrapper).toMatchSnapshot();
});