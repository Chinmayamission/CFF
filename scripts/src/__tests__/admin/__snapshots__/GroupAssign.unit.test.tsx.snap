// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders group assign correctly 1`] = `
<div>
  GA
</div>
`;
